specification/batch/resource-manager/Microsoft.Batch/stable/2021-06-01/BatchManagement.json

```diff
@@ -0,0 +1,5878 @@
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Batch/batchAccounts/{accountName}/pools/{poolName}/stopResize": {
+      "post": {
+        "tags": [
+          "Pool"
+        ],
+        "operationId": "Pool_StopResize",
+        "summary": "Stops an ongoing resize operation on the pool.",
+        "description": "This does not restore the pool to its previous state before the resize operation: it only stops any further changes being made, and the pool maintains its current state. After stopping, the pool stabilizes at the number of nodes it was at when the stop operation was done. During the stop operation, the pool allocation state changes first to stopping and then to steady. A resize operation need not be an explicit resize pool request; this API can also be used to halt the initial sizing of the pool when it is created.",
+        "x-ms-examples": {
+          "StopPoolResize": {
+            "$ref": "./examples/PoolStopResize.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PoolNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation was successful. The response contains the pool entity.",
+            "schema": {
+              "$ref": "#/definitions/Pool"
+            },
+            "headers": {
+              "ETag": {
+                "description": "The ETag HTTP response header. This is an opaque string. You can use it to detect whether the resource has changed between requests. In particular, you can pass the ETag to one of the If-Match or If-None-Match headers.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Batch/batchAccounts/{accountName}/outboundNetworkDependenciesEndpoints": {
```

Comment:
Any resource type MUST implement GET for collection and single resource regardless if it's a singleton or not.
Please add below API 
GET /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Batch/batchAccounts/{accountName}/outboundNetworkDependenciesEndpoints/{endpointName}

Reply:
OutboundNetworkDependenciesEndpoints are not a true customer-owned ARM resource and do not have ids, names, tags etc. This endpoint gives some read-only information about batchAccounts and does not represent a sub-resource of batchAccounts.

Since the individual list items are not resources and do not have ids or names, it is not meaningful to have a /outboundNetworkDependenciesEndpoints/{endpointName} API endpoint.

This endpoint is a Vanguard requirement made according to a Networking spec. It should be noted that many teams are implementing endpoints exactly like this one and some teams have already gotten them approved+merged+released. See AKS at https://github.com/Azure/azure-rest-api-specs/pull/14147/files and a similar dispute at https://github.com/Azure/azure-rest-api-specs/pull/14332#discussion_r629606824

Reply:
Alright, sounds good. Signing off as the same pattern exists across other services.